packages/components/pie-webc-testing/src/playwright/fixtures.ts

```diff
@@ -8,9 +8,9 @@ interface ExtendedTestContext {
 }
 
 // Extend base test by providing "makeAxeBuilder"
-// This new "test" can be used in multiple test files, and each of them will get
+// This new "litTest" can be used in multiple test files, and each of them will get
 // a consistently configured AxeBuilder instance.
-export const test = baseTest.extend<ExtendedTestContext>({
+export const litTest = baseTest.extend<ExtendedTestContext>({
```

Comment:
nit: I think the naming is a bit confusing,  `@sand4rt/experimental-ct-web` is not limited to lit only if I understand correctly. do you think we can keep the const as "test" or maybe have a more generic name such as `WebComponentTest` sorry naming is hard 😆

Reply:
Yeah, I agree, I don't like this implementation, but it's due to the fact we have both

`fixtures.ts` - Provides the makeAxeBuilder functionality to tests using the `@sand4rt/experimental-ct-web` package

`playwright-fixtures.ts` - The exact same file / functionality, but for packages using the native `@playwright/test` package (pie-docs + other consumers).


The reason for the `litTest`, is because we can't export something called `test` twice.

I hope that makes sense...

Reply:
It's also worth noting this is hopefully a short-term hack, as we'll probably be looking to migrate away from this package soon, in favour of testing within Storybook (allowing us to use the native `@playwright/test` package, and therefore, removing the need for this package.